### ISSUE TITLE
Add on-screen keyboard for track selection

### ISSUE THREAD

**Issue by obbardc** [pideck/pideck#1]
Needs to be customisable to show the right keys and easily opened across various languages

https://packages.debian.org/jessie/florence looks like a good fit.

**Comment by danielhjames** [comment]
Florence looks nice, but has a lot of dependencies - e.g. why does a keyboard depend on gstreamer?

**Comment by obbardc** [comment]
Yep! Why is there no on-screen keyboard which just works easily? We don't need per-user config or other expensive blost included.

**Comment by ximae** [comment]
did you try onboard?

not sure how many dependencies does it have though.
